index/index.go

```diff
@@ -38,6 +38,8 @@ const (
 
 	indexFormatV1 = 1
 	indexFormatV2 = 2
+
+	labelNameSeperator = "\xff"
```

Comment:
if you export this, there are view more places where you can use it.

Reply:
Other than few test files, where I see it being used is `labels/labels.go` and `chunkenc/xor.go`.
In both places they are being used as different data type, and also in different context. So I would like to keep this index specific thing unexposed as those other packages are not related to index directly.

I would like to know where did you plan to reuse this.